README.md

```diff
@@ -165,6 +170,7 @@ grunt.initConfig({
     default: {
       options: {
           path: 'test/js/*.html',
+          cwd: 'target/combined-sources',
```

Comment:
This cwd option will break the described actions performed by the `Basic Options`.  It should be removed from this example config I think